app/views/users/show.html.erb

```diff
@@ -1 +1,28 @@
-<h1> User </h1>
\ No newline at end of file
+<section id='user-profile'>
+  <%= link_to '<-- Back to users list', users_url%>
+  <div id='user-card-full'>
+    <h1><%= @user.name %>'s Profile</h1>
+    <div id='user-bio'>
+      <img src=<%= @user.photo %> alt="User <%= @user.id %> picture" width="225" height="225">
+      <p>
+        <%= @user.bio %>
+        <br>
+        I've created <%= @user.posts_counter %> posts in total.
+      </p>
+    </div>
+  </div>
```

Comment:
- [Optional] Kindly make sure that your user show page should match the wireframe below:

![Screenshot from 2022-09-21 21-39-17](https://user-images.githubusercontent.com/63087491/191606601-3186895e-d595-4b19-a9d4-d9dc229677e7.png)

That should go the same with the other pages. You didn't follow the precise design and layout of the wireframes

Reply:
Thanks for the review! I will implement that.